src/fldeff_cut.c

```diff
@@ -21,6 +21,8 @@
 #include "constants/field_effects.h"
 #include "constants/songs.h"
 
+extern bool8 IsMewPlayingHideAndSeek(void);
```

Comment:
super small thing, but this function comes from `include/faraway_island.h`

Reply:
Ah. It was sorta put there during the merge process.